test/router-options.test.js

```diff
@@ -25,7 +25,7 @@ test('Should honor ignoreTrailingSlash option', t => {
   })
 
   fastify.listen({ port: 0 }, (err) => {
-    fastify.server.unref()
+    t.teardown(() => { fastify.close() })
     if (err) t.threw(err)
```

Comment:
calling `t.threw` instead of `t.error` as we do in most other test files.
For consistency, would change to `t.error` and call before `t.teardown`.